app/lib/util/files.js

```diff
@@ -110,35 +116,54 @@ function readFileAsJSON(file) {
 }
 
 /**
- * Find files by pattern in a list of search paths.
+ * Find files matching a given pattern in one or more search paths.
  *
- * @param {Object} options
- * @param {Array<string>} options.searchPaths
- * @param {string} options.pattern
+ * @param {string} pattern
+ * @param {Object} [options]
+ * @param {string[]} [options.searchPaths]
+ * @param {boolean} [options.absolute=true]
+ * @param {boolean} [options.nodir=true]
  *
- * @return {Array<string>} list of found, absolute path names
+ * @return {string[]}
  */
-function globFiles(options) {
+function globFiles(pattern, options = {}) {
+  const defaultOptions = {
+    absolute: true,
+    onlyFiles: true
+  };
 
   const {
     searchPaths,
-    pattern
+    ...otherOptions
```

Comment:
Should this be options?

Reply:
I chose `otherOptions` for the options that will be passed to `globSync`. `searchPaths` should not be included in those options.

Reply:
I missed this declaration, my bad :see_no_evil:

Reply:
I found it simpler to re-use `globFiles` recursively: https://github.com/camunda/camunda-modeler/pull/3863/commits/d65a838f3dbe03fba6ec6dac3fa6eefdbb95d188.

Let me know if this works for you, too.

Reply:
Works for me 👍🏻